proposals/lip-0037.md

```diff
@@ -67,26 +69,37 @@ The tag is chosen depending on the message type. Moreover, different serializati
 
 When a new tag is required in the future, the format:
 
-    "LSK_" || TAG || "_"
+    "LSK_" + TAG + "_"
 
 has to be used where:
 
 - Strings in double quotes are ASCII-encoded literals.
 - TAG is a unique string indicating the scheme and, optionally, additional information, e.g., to specify a version number as shown [above](#message-tags). TAG must contain only ASCII characters between 0x21 and 0x7e (inclusive), except that it must not contain underscore (0x5f).
 
+### Chain Identifiers
+
+Chain identifiers are 4-byte values. The first byte is set to `CHAIN_ID_PREFIX_MAINNET` for chains running in the mainnet network and to `CHAIN_ID_PREFIX_TESTNET` for chains running in the testnet network. The other 3 bytes are unique to that chain.
+
+The following table defines the chain-identifiers prefixes currently specified.
+
+| Name                            | Type  | Value| Description                                                 |
+|---------------------------------|-------|------|-------------------------------------------------------------|
+| `CHAIN_ID_PREFIX_MAINNET`       | bytes | 0x00 | Chain-identifiers prefix for chains running in the mainnet. |
```

Comment:
```suggestion
| `CHAIN_ID_PREFIX_MAINNET`       | bytes | 0x00 | Chain-identifier prefix for mainnet blockchains. |
```

Reply:
"Running in the mainnet" sounds odd.

Reply:
Do we mention here that it will be a requirement to have the prefix `0x00` to connect to Lisk Mainnet for interoperability (I am assuming there that we would make this a requirement) and reference to the chain registration LIP?

Reply:
I thought that this LIP should be unaware of other modules using the chainID for something else. Because otherwise we would also need to add the requirement that all token IDs in the mainnet start with that prefix etc..